src/app/view/components/dot-persona-selector/dot-persona-selector.component.scss

```diff
@@ -0,0 +1,64 @@
+@import "variables";
+
+.dot-persona-selector {
+    .ui-dataview-content.ui-widget-content {
+        width: 448px;
+    }
+
+    .ui-overlaypanel-content {
+        padding: $basic-padding * 2 0;
+    
+        .searchable-dropdown__search {
+            margin: 0 $basic-padding * 2;
+    
+            .searchable-dropdown__search-icon {
+                right: $basic-padding;
+            }
+    
+            .searchable-dropdown__search-inputfield {
+                border: 1px solid $gray-light;
+                border-radius: $border-radius;
+                box-shadow: none;
+                padding: 0 $basic-padding * 4 0 $basic-padding * 2;
+            }
+        }
+    }
+}
+
+.dot-persona-selector__container {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    height: 48px;
+    position: relative;
+    width: 206px;
+
+    dot-avatar {
+        margin-right: $basic-padding * 2;
+    }
+
+    .dot-persona-selector__label-container{
+        flex-grow: 1;
+
+        span {
+            display: block;
+        }
+        .dot-persona-selector__label {
+            color: $gray-light;
+            flex: 1;
+            font-size: $font-size-small;
+        }
+        .dot-persona-selector__name {
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            width: 150px;
```

Comment:
I know we need a width to do the `ellipsis` but in this case we should find a better approach that adjust the to the width of the container.

Reply:
This approach might work: https://css-tricks.com/flexbox-truncated-text/

Reply:
I tried it, but had no luck

Reply:
Check this out: https://jsbin.com/kuwekex/edit?html,css,output